osi_referenceline.proto

```diff
@@ -121,23 +171,24 @@ message ReferenceLine
     // coordinate.
     //
     // Example:
-    // \image html OSI_ReferenceLine1.svg
-    //
-    // This shows a reference line (consisting of three points), and five points
-    // not on the reference line.
-    //
-    // - For \c P1, the situation is clear, since there is exactly one nearest
-    //   point on the polyline. The resulting ST coordinate uniquely maps back
-    //   to \c P1.
-    // - \c P2 has multiple points "nearest points" on the polyline.
-    //   As can be seen here, two  ST coordinates map to \c P2 (red and grey
-    //   dotted line).  Following the rules above, the one with the smallest S
-    //   value is chosen (the red dotted line).
-    // - \c P3 has a unique "nearest point" on the polyline. However, multiple
-    //   points map to the same ST coordinate as that of \c P3, e.g. \c P4
-    //   (drawn in grey).
-    // - Finally, \c P5 shows how the reference line is extended infinitely for
-    //   points that are "outside" the reference line.
+    // \image html OSI_ReferenceLine1.svg "S, T calculation"
```

Comment:
This now references the new image. But if the old mode is to be kept, then the old image also needs to stay, I think (plus the documentation that accompanied the image). Otherwise it is unclear how to implement the old mode, and how to interpret it.